src/types/trail.ts

```
@@ -0,0 +1,69 @@
+
+export interface GeoJSONFeature {
+  type: 'Feature';
+  geometry: GeoJSONPoint | GeoJSONLineString;
+  properties: {
+    id?: string;
+    name?: string;
+    courseType?: string;
+    difficulty?: string;
+    distance?: number;
+    duration?: number;
+    [key: string]: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)